superset-frontend/src/views/CRUD/welcome/Welcome.tsx

```diff
@@ -89,21 +87,15 @@ function Welcome({ user, addDangerToast }: WelcomeProps) {
   const recent = `/superset/recent_activity/${user.userId}/?limit=6`;
   const [activeChild, setActiveChild] = useState('Viewed');
   const [activityData, setActivityData] = useState<ActivityData>({});
+  const [chartData, setChartData] = useState<Array<object> | null>(null);
+  const [queryData, setQueryData] = useState<Array<object> | null>(null);
+  const [dashboardData, setDashboardData] = useState<Array<object> | null>(null);
   const [loading, setLoading] = useState(true);
+
   useEffect(() => {
     getRecentAcitivtyObjs(user.userId, recent, addDangerToast)
       .then(res => {
-        const data: any = {
-          Created: [
-            ...res.createdByChart,
-            ...res.createdByDash,
-            ...res.createdByQuery,
-          ],
-          myChart: res.createdByChart,
-          myDash: res.createdByDash,
-          myQuery: res.createdByQuery,
-          Edited: [...res.editedChart, ...res.editedDash],
-        };
+        const data: any = {};
```

Comment:
Not a dealbreaker, since it's not the intent of this PR, but it would be nice to add stronger typing to any `any` or `object` types in this area of the code. Maybe track it for a follow-up PR if it's too much feature creep for this one.

Reply:
done